frontend/src/app/modules/availability/new-slot/new-availability/new-availability.component.ts

```diff
@@ -58,4 +58,12 @@ export class NewAvailabilityComponent implements OnInit {
             this.slot.isEnabled = !this.slot.isEnabled;
         }
     }
+
+    public saveChanges() {
+        if (this.index === 0 && this.generalComponent.generalForm.invalid) {
```

Comment:
Why do you check the index here?

Reply:
To allow to "save changes" on other tabs even if there is invalid data on the general tab left

Reply:
Do we need this? we save all data in one request so data in all tabs should be valid

Reply:
Ok, I agree